### PR TITLE
New Addition and Existing Changes

### PR DESCRIPTION
**Pull request by imisterk**
Added new individual portfolio.
Changed incorrect links under Creative Agencies.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,15 +94,17 @@
 - [Nick Jones](http://www.narrowdesign.com/)
 - [Samuel Ramos](http://www.highfivebro.com/)
 - [Rick Waalders](http://rickwaalders.com/)
+- [Jean-Baptiste Giffard](http://www.digitaldesigner.cool/)
```
Comment (TheJaredWilcurt) [inline README.md]: 👍

File: README.md [hunk 2]
```diff
@@ -94,15 +94,17 @@
...
 ## Creative Agencies
 - [Bluegg](https://bluegg.co.uk/)
 - [Humaan](https://humaan.com/)
 - [Creative Navy](https://www.interface-design.co.uk/)
 - [Artists Web](https://www.artistsweb.cz/)
-- [Strange Love](http://pupcreativeagency.com/)
+- [Strange Love](http://strangelove.co/)
 - [First Born](https://www.firstborn.com/)
 - [Pup Creative Agency](http://pupcreativeagency.com/)
+- [Lobods](https://lobods.com/en/)
```
Comment (TheJaredWilcurt) [inline README.md]: 👍

File: README.md [hunk 3]
```diff
@@ -94,15 +94,17 @@
...
 - [Rick Waalders](http://rickwaalders.com/)
+- [Jean-Baptiste Giffard](http://www.digitaldesigner.cool/)
 
 ## Creative Agencies
 - [Bluegg](https://bluegg.co.uk/)
 - [Humaan](https://humaan.com/)
 - [Creative Navy](https://www.interface-design.co.uk/)
 - [Artists Web](https://www.artistsweb.cz/)
-- [Strange Love](http://pupcreativeagency.com/)
+- [Strange Love](http://strangelove.co/)
```
Comment (TheJaredWilcurt) [inline README.md]: dead link

Comment (imisterk) [reply README.md]: Yeah looks like they have recently shut down or not renewed their domain. Unless they changed it to - https://strangelove.nl/ which I don't think can make this list.

**Pull request merged**